File: NumbersGenerator/numb.c
#include <stdio.h>
#include <time.h>

int main(){
    long start = time(NULL);
    int count = 0;
    while(count<=100000){
        printf("%d\n", count);
        count++;
    }
    long end = time(NULL);
    int time = (end-start); 

    printf("\nCurrent time: %d seconds.", time);
    return 0;
}

File: NumbersGenerator/numb.py
import time
def numb():
    start = time.time() #Starts time
    count = 0
    while count <= 100000:
        print(count)
        count += 1

    end = time.time() #Ends time
    stime = end - start #Sets time
    print("\nCurrent time: " + str(stime) + " seconds.")

if __name__ == '__main__':
    numb()